gp-includes/assets-loader.php

```diff
@@ -1,84 +1,94 @@
 <?php
 /**
- * Defines default styles and scripts
+ * Defines default styles and scripts.
+ *
+ * @package GlotPress
+ * @since 1.0.0
  */
 
 /**
- * Register the GlotPress styles
- *
- * @param WP_Styles $styles
+ * Register the GlotPress styles and load the base style sheet.
  */
-function gp_styles_default( &$styles ) {
+function gp_styles_default() {
 	$url = gp_plugin_url( 'assets/css' );
 
-	$styles->add( 'base', $url . '/style.css', array(), '20150717' );
+	// Register our base style.
+	wp_register_style( 'gp-base', $url . '/style.css', array(), '20150717' );
 }
 
-add_action( 'wp_default_styles', 'gp_styles_default' );
+add_action( 'init', 'gp_styles_default' );
 
 /**
- * Register the GlotPress scripts
- *
- * @param WP_Scripts $scripts
+ * Register the GlotPress scripts.
  */
-function gp_scripts_default( &$scripts ) {
+function gp_register_scripts() {
 	$url = gp_plugin_url( 'assets/js' );
 
-	$scripts->add( 'tablesorter', $url . '/jquery.tablesorter.min.js', array( 'jquery' ), '1.10.4' );
-
-	$scripts->add( 'gp-common', $url . '/common.js', array( 'jquery' ), '20150430' );
-	$scripts->add( 'gp-editor', $url . '/editor.js', array( 'gp-common', 'jquery-ui-tooltip' ), '20160329' );
-	$scripts->add( 'gp-glossary', $url . '/glossary.js', array( 'gp-common' ), '20160329' );
-	$scripts->add( 'gp-translations-page', $url . '/translations-page.js', array( 'gp-common' ), '20150430' );
-	$scripts->add( 'mass-create-sets-page', $url . '/mass-create-sets-page.js', array( 'gp-common' ), '20150430' );
+	// Register our standard scripts.
+	wp_register_script( 'tablesorter', $url . '/jquery.tablesorter.min.js', array( 'jquery' ), '1.10.4' );
+	wp_register_script( 'gp-common', $url . '/common.js', array( 'jquery' ), '20150430' );
+	wp_register_script( 'gp-editor', $url . '/editor.js', array( 'gp-common', 'jquery-ui-tooltip' ), '20160329' );
+	wp_register_script( 'gp-glossary', $url . '/glossary.js', array( 'gp-common', 'gp-editor' ), '20160329' );
+	wp_register_script( 'gp-translations-page', $url . '/translations-page.js', array( 'gp-common', 'gp-editor' ), '20150430' );
+	wp_register_script( 'gp-mass-create-sets-page', $url . '/mass-create-sets-page.js', array( 'gp-common', 'gp-editor' ), '20150430' );
```

Comment:
> in the end I decided it was better to be verbose in the dependencies instead of nesting them.

Well, if you want to go this route you have to define 'jquery' too. :)